packages/synapse-react-client/demo/containers/playground/WidgetDemo.tsx

```diff
@@ -92,31 +83,6 @@ export const WidgetDemo: React.FunctionComponent = () => {
 
   return (
     <div className="container">
-      <div style={{ fontWeight: 'bold' }}></div>
```

Comment:
Remove deleted components from deprecated demo (I'll make another PR to just delete this whole thing)